fontforge/bezctx_ff.c

```diff
@@ -1,28 +1,52 @@
-/* This file written by George Williams to provide a gateway to fontforge */
-/* it it a modification of Raph's bezctx_ps.c */
+/*
+Copyright: 2007 Raph Levien
+License: GPL-2+
+    bezctx_ps.c - ppedit - A pattern plate editor for Spiro splines.
+Copyright: 2007 George Williams
+    bezctx_ff.c - Provide a gateway into FontForge with added modification.
+Contributions: Yuri Chornoivan, Jonathan Hanna, Joe Da Silva, Reuben Thomas,
+    Franke-Trampe
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+as published by the Free Software Foundation; either version 2
+of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+02110-1301, USA.
+
+*/
 #include <basics.h>
 #include <stdio.h>
 
 #ifndef _NO_LIBSPIRO
 #include "bezctx_ff.h"
-#include "fontforgevw.h"	/* For LogError, else splinefont.h */
+#include "fontforgevw.h" /* For LogError, else splinefont.h */
```

Comment:
Could we just fix the header and not mess with the whitespace and minor verbiage? I don't think it's worth the changelog/blame clutter.